.builds/linux.yml

```diff
@@ -18,13 +19,23 @@ environment:
 tasks:
   - rustup: |
       curl https://sh.rustup.rs -sSf | sh -s -- -y --default-toolchain stable --profile minimal
-  - test: |
-      cd alacritty
-      cargo test
   - rustfmt: |
       cd alacritty
       rustup toolchain install nightly -c rustfmt
       cargo +nightly fmt -- --check
+  - man-pages: |
+      cd alacritty
+      printf "Compiling alacritty.1.scd" > /dev/null
+      scdoc < extra/man/alacritty.1.scd > /dev/null
+      printf "Compiling alacritty-msg.1.scd" > /dev/null
+      scdoc < extra/man/alacritty-msg.1.scd > /dev/null
+      printf "Compiling alacritty.5.scd" > /dev/null
+      scdoc < extra/man/alacritty.5.scd > /dev/null
+      printf "Compiling alacritty-bindings.5.scd" > /dev/null
+      scdoc < extra/man/alacritty-bindings.5.scd > /dev/null
+  - test: |
+      cd alacritty
+      cargo test
```

Comment:
Is there a reason why test was moved after rustfmt?

Reply:
to make faster jobs before, so they fail sooner.